tests/tracer/telemetry/test_telemetry.py

```diff
@@ -9,7 +9,7 @@
 
 @pytest.fixture
 def telemetry_writer():
-    yield telemetry.telemetry_writer
+    yield telemetry.TelemetryWriter()
```

Comment:
A heartbeat events is queued 60 seconds after `TelemetryWriter()` is initialized.  Using the global telemetry writer causes `test_enable` to become fail (it queues a heartbeat event which is not expected on start up).